.buildkite/integration.pipeline.yml

```
@@ -179,7 +179,11 @@ steps:
       - int-packaging
     command: "buildkite-agent pipeline upload .buildkite/bk.integration.pipeline.yml"
 
-  - label: "Triggering custom FIPS integration tests"
+# Temporarily disabled FIPS integration tests while we're encountering deployment creation
+# issues in the FRH GovCloud Staging environment.
+# Example: https://buildkite.com/elastic/elastic-agent-extended-testing/builds/10211#01995752-0966-4c21-874c-0894305ac091
+  - if: false
```

I wonder if we need a comparison instead ... 

```suggestion
  - if: false != true
```

However, the [docs](https://buildkite.com/docs/pipelines/configure/conditionals#variable-and-syntax-reference-operator-syntax) state:

> The following expressions are supported by the if attribute.
> ....
> Literals | true false null

---

Okay, let's try it. 🤷